includes/Stories_Script_Data.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Stories script data class.
+ *
+ * Stories data which will be required by various JS scripts.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Class Stories_Script_Data.
+ *
+ * @package Google\Web_Stories
+ */
+class Stories_Script_Data {
```

Comment:
Would this make more sense as a trait?

Reply:
Yes, converted to trait.

Reply:
@spacedmonkey @ankitrox I think it worked better as a class. Let's discuss pros/cos before making such changes right away next time :)